cairo/tests/utils/args_gen.py

```diff
@@ -394,6 +394,22 @@ def encode_account(raw_account_data: Account, storage_root: Bytes) -> Bytes:
     )
 
 
+# TODO PR in EELS?
```

Comment:
why PR EELS?

Reply:
instead of doing a patch we could simply PR EELS and avoid that patch

Reply:
do you mean PR EELS to add storage root and codehash in account clasS?

Reply:
no just do account != EMPTY_ACCOUNT